src/components/formik-toggle/FormikToggle.tsx

```diff
@@ -19,29 +21,27 @@ type Props = {
   icon?: any;
 };
 
-const styleVariables = loadStyleVariables();
-
 const ToggleInput = styled.input<{}>`
-  z-index: ${styleVariables.ziCheckbox};
+  z-index: var(--zi-x);
   opacity: 0;
   width: 40px;
   height: 15px;
   position: absolute;
   &:checked ~ label span {
     transform: translatex(30px);
-    transition: transform ${styleVariables.transitionspeedNormal} ease-in;
+    transition: transform var(--transition-speed-normal) ease-in;
   }
   &:checked ~ label {
-    background: ${styleVariables.colorGreen};
-    border: 2px solid ${styleVariables.greenDark};
-    transition: transform ${styleVariables.transitionspeedNormal} ease-in;
+    background: ${color.getColor('primary')};
```

Comment:
FYI: je kan ook gewoon var(--color-primary) gebruiken. De getColor helper heb ik gewoon gemaakt voor componentjes met een color-prop. Bv. ```<Button color='primary'>```.
Dan kan je in de code ```getColor(this.props.color)``` doen ipv ```'var(--color' + this.props.color + ')';```, en hoeft je niet ```<Button color='var(--color-primary)'>``` te doen.

Hoe je het hier gebruikt werkt het ook en is ook ok. Laat het zo gerust maar staan.